.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+Hello dear WTForms community. I have a problem!
```

Comment:
As nice as this is, it's just going to cause noise most of the time. We can use the more powerful template description metadata to add descriptions of what to use the template for, in a future PR.